package.json

```diff
@@ -94,6 +94,7 @@
     "redlock": "^4.2.0",
     "subscriptions-transport-ws": "^0.11.0",
     "twilio": "^3.65.0",
+    "uuid": "^8.3.2",
```

Comment:
this library was removed recently, please use `crypto` package

Reply:
yup, we're aiming at reducing the number of library we depend on